### DISCUSSION TITLE
13.1 is there a way to limit the size of detected areas

### DISCUSSION THREAD

**Discussion by ediguidedog** [blakeblackshear/frigate#9667] [Ask A Question]
The attached probably describes the situation better than words.  On the left is 13.1 on the right 12.0. Its very windy today and I'm seeing a lot of movement in the plants out the front and whereas v12 sees this movement as one large area V13 sees it as dozens of small areas and is slowing right down and dropping detection frames as it struggles to keep up.  I can't put a motion mask in place as it would stop people's upper body being spotted as they approach the zone so is there anything I can do so tell frigate to ignore small movements or at least treat overlapping areas as one large area?
Both versions are running simultaneously on identical hardware but one is using a coral and the other using gpu (normally around 28ms inference) . The images below are of the same point in time.

![Capture](https://github.com/blakeblackshear/frigate/assets/73383390/a2d099a7-1706-47d6-8dbc-8381b870e420)

**Comment by NickM-27** [comment]
> is there anything I can do so tell frigate to ignore small movements

Increase contour area

**Reply by ediguidedog** [reply]
Thanks I'll give that a try. Is this down to the new detection algorithms?

**Reply by NickM-27** [reply]
yes
